components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/IdentityEventConstants.java

```diff
@@ -202,7 +204,8 @@ private EventProperty(){}
         public static final String SESSION_CONTEXT = "sessionContext";
         public static final String PARAMS = "params";
         public static final String AUTHENTICATION_STATUS = "authenticationStatus";
-
+        public static final String USER_CHALLENGE_ANSWERS = "userChallengeAnswers";
+        public static final String USER_OLD_CHALLENGE_ANSWERS = "userNewChallengeAnswers";
```

Comment:
Shall we change the identifier to match the value?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2919/commits/ea7af90737d21f62569a67c50cfdc58d7f96e025